selfdrive/car/tests/test_fw_fingerprint.py

```diff
@@ -183,20 +184,30 @@ def _run_thread(thread: threading.Thread) -> float:
     params.put_bool("ObdMultiplexingEnabled", True)
     thread.start()
     t = time.perf_counter()
+
     while thread.is_alive():
-      time.sleep(0.02)
+      time.sleep(0.001)
       if not params.get_bool("ObdMultiplexingChanged"):
         params.put_bool("ObdMultiplexingChanged", True)
     return time.perf_counter() - t
 
+  def fake_get_data(self, timeout):
+    self.fake_timeout_time += timeout
+    return {}
+
   def _benchmark_brand(self, brand, num_pandas):
+    query_patch = mock.patch("openpilot.selfdrive.car.isotp_parallel_query.IsoTpParallelQuery.get_data", self.fake_get_data)
+    query_patch.start()
+
```

Comment:
I didn't realize it dynamically adds whatever timeout IsoTpParallelQuery, can we get this in now, if it will improve reliability and allow us to lower the threshold again

Reply:
sure I'll reopen this